### ISSUE TITLE
ScreenSaver Volume

### ISSUE THREAD

**Issue by goyometeojorito** [fieldofcows/EmulationStation#21]
I need set volume to 10% when use the option screen saver with random video.
¿This is posible?

**Comment by Darknior** [comment]
No, i have ask for this too, it will be add in a near future ;)

**Comment by Darknior** [comment]
now it's done on the master release, i think you can close this issue
